cmd/bundle/init.go

```diff
@@ -18,9 +20,52 @@ var gitUrlPrefixes = []string{
 	"git@",
 }
 
-var aliasedTemplates = map[string]string{
-	"mlops-stack":  "https://github.com/databricks/mlops-stacks",
-	"mlops-stacks": "https://github.com/databricks/mlops-stacks",
+type nativeTemplate struct {
+	gitUrl      string
+	description string
+	aliases     []string
+}
+
+var nativeTemplates = map[string]nativeTemplate{
+	"default-python": {
+		description: "The default Python template",
+	},
+	"mlops-stacks": {
+		gitUrl:      "https://github.com/databricks/mlops-stacks",
+		description: "The Databricks MLOps Stacks template. More information can be found at: https://github.com/databricks/mlops-stacks",
+		aliases:     []string{"mlops-stack"},
+	},
+}
+
+func nativeTemplateDescriptions() string {
+	var lines []string
+	for name, template := range nativeTemplates {
+		lines = append(lines, fmt.Sprintf("- %s: %s", name, template.description))
+	}
+	return strings.Join(lines, "\n")
+}
+
+func nativeTemplateOptions() []string {
+	keys := make([]string, 0, len(nativeTemplates))
+	for k := range nativeTemplates {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+	return keys
+}
+
+func getURLForNativeTemplate(name string) string {
```

Comment:
Mixed casing: `URL` here and `Url` in the field.

Reply:
Done. Also added test coverage.